transformer_engine/pytorch/attention.py

```diff
@@ -1823,6 +1823,8 @@ def forward(ctx, is_training, max_seqlen, cu_seqlens, qkv, qkv_dtype, attn_bias,
 
     @staticmethod
     def backward(ctx, d_out):
+        if not d_out.is_contiguous():
+            d_out = d_out.contiguous()
```

Comment:
`.contiguous` is a no-op if the data is already contiguous:
```suggestion
        d_out = d_out.contiguous()
```